linux_os/guide/services/ntp/service_ntpd_enabled/rule.yml

```diff
@@ -24,7 +24,7 @@ identifiers:
 
 references:
     disa@rhel6: "160"
-    nist: AU-8(1)
+    nist: AU-8(1),AU-12(1)
```

Comment:
```suggestion
    nist: CM-6(a),CM-6(b)
```

Reply:
Updating to CM is out of scope of this PR.